setup.py

```diff
@@ -39,6 +39,7 @@ def parse_requirements_file (filename: str) -> typing.List:
 
     base_packages = parse_requirements_file("requirements.txt")
     docs_packages = parse_requirements_file("requirements-dev.txt")
+    viz_packages = parse_requirements_file("requirements-viz.txt")
```

Comment:
I would have added just 
```suggestion
    viz_packages = ["altair"]
```
but I saw the requirements-dev.txt file so I thought keeping it consistent was good.

What I do usually is that I parse the `requirements.txt` file for base packages (this way I use it in `setup.py` and in `Dockerfile` when docker is relevant) but for `dev`, `docs`, `viz`, ... I usually use arrays directly.